k8s/vagrant/README.md

```diff
@@ -123,6 +123,13 @@ Starting to serve on 127.0.0.1:8001
 ### Verify 
 
 Launch the URL http://127.0.0.1:8001/ui
+While accessing above URL if you see following Error.
```

Comment:
While accessing the above URL, if you see the following error, launch the URL http....